libraries/helpers.rb

```diff
@@ -354,7 +354,7 @@ def url_for(pkg, base_url = 'ftp://ftp.software.ibm.com/aix/freeSoftware/aixtool
           'xft' => { version: '2.1.6-5', rpm: 'xft/xft-2.1.6-5.aix5.1.ppc.rpm' },
           'xpdf' => { version: '1.00-1', rpm: 'xpdf/xpdf-1.00-1.aix4.3.ppc.rpm' },
           'xpm-devel' => { version: '3.4k-8', rpm: 'xpm/xpm-devel-3.4k-8.aix6.1.ppc.rpm' },
-          'xpm' => { version: '3.4k-8', rpm: 'xpm/xpm-3.4k-8.aix5.2.ppc.rpm' },
+          # 'xpm' => { version: '3.4k-8', rpm: 'xpm/xpm-3.4k-8.aix5.2.ppc.rpm' },
```

Comment:
I still had a terminal with the offending output, here's a snippet of what I was seeing (in rspec):

```shell
tests::remote_chef_gem
  aix 6.1
/tmp/chefspec20161104-26989-lxarfile_cache_path/cookbooks/aix/libraries/helpers.rb:357: warning: key "xpm" is duplicated and overwritten on line 358
Please report a bug if this causes problems.
    cleanup dummies
/tmp/chefspec20161104-26989-lxarfile_cache_path/cookbooks/aix/libraries/helpers.rb:357: warning: key "xpm" is duplicated and overwritten on line 358
    downloads and installs dummy gem
/tmp/chefspec20161104-26989-lxarfile_cache_path/cookbooks/aix/libraries/helpers.rb:357: warning: key "xpm" is duplicated and overwritten on line 358
    downloads and upgrades dummy gem
  aix 7.1
/tmp/chefspec20161104-26989-lxarfile_cache_path/cookbooks/aix/libraries/helpers.rb:357: warning: key "xpm" is duplicated and overwritten on line 358
    cleanup dummies
/tmp/chefspec20161104-26989-lxarfile_cache_path/cookbooks/aix/libraries/helpers.rb:357: warning: key "xpm" is duplicated and overwritten on line 358
    downloads and installs dummy gem
/tmp/chefspec20161104-26989-lxarfile_cache_path/cookbooks/aix/libraries/helpers.rb:357: warning: key "xpm" is duplicated and overwritten on line 358
    downloads and upgrades dummy gem
  redhat 7.0
/tmp/chefspec20161104-26989-lxarfile_cache_path/cookbooks/aix/libraries/helpers.rb:357: warning: key "xpm" is duplicated and overwritten on line 358
    cleanup dummies
/tmp/chefspec20161104-26989-lxarfile_cache_path/cookbooks/aix/libraries/helpers.rb:357: warning: key "xpm" is duplicated and overwritten on line 358
    downloads and installs dummy gem
/tmp/chefspec20161104-26989-lxarfile_cache_path/cookbooks/aix/libraries/helpers.rb:357: warning: key "xpm" is duplicated and overwritten on line 358
    downloads and upgrades dummy gem
```